docs/guides/nrfconnect_examples_software_update.md

```diff
@@ -236,93 +245,93 @@ Complete the following steps to perform DFU using mcumgr:
+        version: 0.0.0
+        bootable: true
+        flags:
+        hash: d9e31e73cb7a959c26411250c2b3028f3510ae88a4549ae3f2f097c3e7530f48
+    Split status: N/A (0)
+    ```
 
     c. Swap the firmware images by calling the following method with
     `image-hash` replaced by the image present in the slot 1 hash (for example,
     `d9e31e73cb7a959c26411250c2b3028f3510ae88a4549ae3f2f097c3e7530f48`):
 
-         ```
-         sudo mcumgr --conntype ble --hci ble-hci-number --connstring peer_name='ble-device-name' image test image-hash
-         ```
+    ```
+    sudo mcumgr --conntype ble --hci ble-hci-number --connstring peer_name='ble-device-name' image test image-hash
+    ```
 
     You can observe that the `flags:` field in the image for slot 1 changes
     value to `pending`:
 
-         ```
-         Images:
-         image=0 slot=0
-             version: 0.0.0
-             bootable: true
-             flags: active confirmed
-             hash: 7bb0e909a846e833465cbb44c581cf045413a5446c6953a30a3dcc2c3ad51764
-         image=0 slot=1
-             version: 0.0.0
-             bootable: true
-             flags: pending
-             hash: cbd58fc3821e749d3abfb00b3069f98c078824735f1b2a333e8a1579971e7de1
-         image=1 slot=1
-             version: 0.0.0
-             bootable: true
-             flags: pending
-             hash: d9e31e73cb7a959c26411250c2b3028f3510ae88a4549ae3f2f097c3e7530f48
-         Split status: N/A (0)
-         ```
+    ```
+    Images:
+    image=0 slot=0
+        version: 0.0.0
+        bootable: true
+        flags: active confirmed
+        hash: 7bb0e909a846e833465cbb44c581cf045413a5446c6953a30a3dcc2c3ad51764
+    image=0 slot=1
+        version: 0.0.0
+        bootable: true
+        flags: pending
+        hash: cbd58fc3821e749d3abfb00b3069f98c078824735f1b2a333e8a1579971e7de1
+    image=1 slot=1
+        version: 0.0.0
+        bootable: true
+        flags: pending
+        hash: d9e31e73cb7a959c26411250c2b3028f3510ae88a4549ae3f2f097c3e7530f48
+    Split status: N/A (0)
+    ```
 
 8.  Reset the device with the following command to let the bootloader swap
     images:
 
-        ```
-        sudo mcumgr --conntype ble --hci ble-hci-number --connstring peer_name='ble-device-name' reset
-        ```
+    ```
+    sudo mcumgr --conntype ble --hci ble-hci-number --connstring peer_name='ble-device-name' reset
+    ```
 
 The device is reset and the following notifications appear in its console:
 
-       ```
-       *** Booting Zephyr OS build zephyr-v2.5.0-1101-ga9d3aef65424  ***
-       I: Starting bootloader
-       I: Primary image: magic=good, swap_type=0x2, copy_done=0x1, image_ok=0x1
-       I: Secondary image: magic=good, swap_type=0x2, copy_done=0x3, image_ok=0x3
-       I: Boot source: none
-       I: Swap type: test
-       ```
+```
+*** Booting Zephyr OS build zephyr-v2.5.-1101-ga9d3aef65424  ***
+I: Starting bootloader
+I: Primary image: magic=good, swap_type=0x2, opy_done=0x1, image_ok=0x1
```

Comment:
@greg-fer are you sure about that?
https://developer.nordicsemi.com/nRF_Connect_SDK/doc/2.3.0/nrf/app_dev/bootloaders_and_dfu/bootloader_testing.html (MCUboot tab )

Reply:
Yes, @markaj-nordic , I wonder more where the edits inside the snippet came from. Let me fix those.